### PR TITLE
added browsers, browser elements, and advanced guides

### PR DESCRIPTION
**Pull request by thekiiingbob**
Added the Browsers, Browser Elements, and Advanced Interactions sections from watirwebdriver.com. They can be accessed via the guides page, which is available in the header.

### CODE REVIEW

**Comment by thekiiingbob** [comment]
I also fixed the link on the chrome.md page in reference to https://github.com/watir/watir-webdriver/issues/404

File: _docs/browser-certificates.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+{% endhighlight %}
+
+The reason this is needed is explained [here](http://code.google.com/p/selenium/wiki/UntrustedSSLCertificates#Implementation_details).
+
+### Chrome
+
+It is easy to ignore invalid Browser certificates in Google Chrome by passing a command line switch:
+
+{% highlight ruby %}
+Watir::Browser.new :chrome, :switches => ['--ignore-certificate-errors']
```
Comment (p0deje) [inline _docs/browser-certificates.md]: As far as I remember, it's already set by default by chromedriver.

Also, as the general idea - let's not use Ruby 1.8 in examples (i.e. `:switches =>` should be `switches:`).

File: _docs/browser-certificates.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+If you have a trusted certificate, but there is some other certificate error such as a hostname mismatch (eg. using a production certificate in test), you should do the following:
+
+{% highlight ruby %}
+profile = Selenium::WebDriver::Firefox::Profile.new
+profile.assume_untrusted_certificate_issuer = false
+b = Watir::Browser.new :firefox, :profile => profile
+{% endhighlight %}
+
+The reason this is needed is explained [here](http://code.google.com/p/selenium/wiki/UntrustedSSLCertificates#Implementation_details).
```
Comment (p0deje) [inline _docs/browser-certificates.md]: Selenium is no longer on Google Code, so the link should be https://github.com/SeleniumHQ/selenium/wiki/Untrusted-SSL-Certificates#implementation-details

**Comment by thekiiingbob** [comment]
I addressed @p0deje concerns. There is probably still plenty of stuff that needs to be updated, but we can at least get this in and then start updating the content.

**Pull request merged**